moped-editor/src/views/projects/projectView/ProjectComments.js

```diff
@@ -79,7 +79,7 @@ const useStyles = makeStyles((theme) => ({
     marginTop: "25px",
   },
   editDeleteButtons: {
-    color: "#000000",
+    color: theme.palette.text.primary,
```

Comment:
Unrelated to the fix but replaced this hex in the wild 🎨

Reply:
thank you for taming this wild hex!